doc/admin-guide/files/remap.config.en.rst

```diff
@@ -675,18 +767,18 @@ PURGE  \-             \-             \-             deny             denied  (40
 PUSH   \-             \-             \-             deny             denied  (403)
 ====== ============== ============== ============== ================ =============
 
-With the "Match on IP only Policy", the in-line filter works like an :file:`ip_allow.yaml` rule applies to all requests to
-``www.example.com`` that denies ``POST`` requests and implicitly allows all other methods:
+With the match on IP only policy, the in-line filter works like an :file:`ip_allow.yaml` rule: it applies to all requests to
+``www.example.com`` and denies ``POST`` requests while allowing all other methods:
 
 ====== ================ ============== ============== ============= =============
 Method In-line Filter   Named Filter 1 Named Filter 2 ip_allow.yaml result
 ====== ================ ============== ============== ============= =============
-GET    allow (implicit) \-             \-             \-            allowed (200)
+GET    allow            \-             \-             \-            allowed (200)
 POST   deny             \-             \-             \-            denied  (403)
-HEAD   allow (implicit) allow          \-             \-            allowed (200)
-DELETE allow (implicit) \-             deny           \-            allowed (200)
-PURGE  allow (implicit) \-             \-             deny          allowed (200)
-PUSH   allow (implicit) \-             \-             deny          allowed (200)
+HEAD   allow            \-             \-             \-            allowed (200)
+DELETE allow            \-             \-             \-            allowed (200)
+PURGE  allow            \-             \-             \-            allowed (200)
+PUSH   allow            \-             \-             \-            allowed (200)
```

Comment:
Getting rid of (implicit) is good, but would you leave actions in the following filter to make sure "In-line Filter" wins than others.